g2p_registry_individual/models/registry_config.py

```diff
@@ -0,0 +1,25 @@
+# Part of OpenG2P. See LICENSE file for full copyright and licensing details.
+
+from odoo import api, fields, models
+
+
+class RegistryConfig(models.TransientModel):
+    _inherit = "res.config.settings"
+
+    gender_config = fields.Char(store=True)
+
+    def set_values(self):
+        res = super(RegistryConfig, self).set_values()
+        params = self.env["ir.config_parameter"].sudo()
+        params.set_param("g2p_registry.gender_config", self.gender_config)
+        return res
+
+    @api.model
+    def get_values(self):
+        res = super(RegistryConfig, self).get_values()
+        res.update(
+            gender_config=self.env["ir.config_parameter"]
+            .sudo()
+            .get_param("g2p_registry.gender_config"),
+        )
+        return res
```

Comment:
I think this should also be accompanied with a data XML, which defines the default value of gender_config. (Depending on how the gender_config parameter value is decided). For example:
```
<?xml version="1.0" encoding="utf-8" ?>
<odoo>
    <function
        model="ir.config_parameter"
        name="set_param"
        eval="('g2p_registry.gender_config', '[("male","Male"),("female","Female")]')"
    />
</odoo>
```